### PR TITLE
ui(Show-meet-details):Added Ui to show meet details

### PR DESCRIPTION
**Pull request by Aditya3034**
### Functionality:
Created UI to show meet details.
### Solution:
added show-meet-details component and did changes in meet component
### Risk level:
- [ ] high 
- [x] medium
- [ ] low

### How to test:
Steps to check:-
1.Open emulator
2. click on meet icon.
3. schedule meet
4. You can see the meet details below that.

### CODE REVIEW

File: functions/model/meet/tark/getMeetDetails.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+/* eslint-disable linebreak-style */
+
+/** *********************************************************
+ * Copyright (C) 2022
+ * Worktez
+ * Author: Simran Nigam <nigamsimran14@gmail.com>
```
Comment (sanjaykrishna1203) [inline functions/model/meet/tark/getMeetDetails.js:6]: Change the Author name

File: src/app/header/show-meet-details/show-meet-details.component.ts [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+@Component({
+  selector: 'app-show-meet-details',
+  templateUrl: './show-meet-details.component.html',
+  styleUrls: ['./show-meet-details.component.css']
+})
+export class ShowMeetDetailsComponent implements OnInit {
+
+  meetData:Meet[]=[];
+
+  @Output() getMeet: EventEmitter<string> = new EventEmitter();
```
Comment (sanjaykrishna1203) [inline src/app/header/show-meet-details/show-meet-details.component.ts:20]: No need of Event emitter here, same for the Input

**Review by sanjaykrishna1203 — CHANGES_REQUESTED** [review]

File: src/app/header/show-meet-details/show-meet-details.component.ts (hunk 2 above)
Comment (Aditya3034) [reply src/app/header/show-meet-details/show-meet-details.component.ts]: Did the reqquired changes.

Comment (Aditya3034) [reply src/app/header/show-meet-details/show-meet-details.component.ts]: did the required changes.

**Review by viv9k — APPROVED** [review]
Approved

**Pull request merged**